thunder/benchmarks/neva.py

```diff
@@ -0,0 +1,6275 @@
+import pytest
+import torch
+import thunder
+import einops
+
+
+@pytest.mark.benchmark(
+    group="neva",
+    warmup_iterations=1,  # takes a long time to run.
+)
+def g1():
+    class DynamoModule(torch.nn.Module):
+        def forward(
+            self,
+            L_vision_x_: torch.Tensor,
+            L_self_modules_vision_encoder_modules_vision_model_modules_embeddings_parameters_class_embedding_: torch.nn.parameter.Parameter,
```

Comment:
Using a shorter prefix instead of `L_self_modules_vision_encoder_modules_vision_model_modules` would make the code more readable.